runtime/moonbase/src/xcm_config.rs

```diff
@@ -149,8 +175,72 @@ pub type LocalAssetTransactor = XcmCurrencyAdapter<
 	// We dont allow teleport
 	(),
 >;
+
+/// Means for transacting local assets that are not the native currency
+/// This transactor uses the old reanchor logic
+/// Remove once we import https://github.com/open-web3-stack/open-runtime-module-library/pull/708
+pub type LocalFungiblesTransactorOldReanchor = FungiblesAdapter<
```

Comment:
"Old" and "New" make sense now, but they will have lost their context in the future :) Not a big deal if they are temporary...

Reply:
I am planning to remove them as soon as we release 1400 so I think that's fine